app/src/main/java/com/hedvig/app/ui/compose/composables/buttons/PrimaryButtons.kt

```diff
@@ -33,29 +51,24 @@ fun LargeContainedButton(
 }
 
 @Preview(
-    name = "LargeContainedButton",
-    group = "Design System",
+    name = "PrimaryButton",
+    group = "Buttons",
 )
 @Composable
-fun LargeContainedButtonPreview() {
+fun PrimaryButtonPreview() {
     HedvigTheme {
-        LargeContainedButton({}) {
-            Text("Contained Button (Large)")
-        }
+        PrimaryTextButton(text = "Contained Button (Large)") {}
     }
 }
 
-@ShowkaseComposable(skip = true)
```

Comment:
We probably do want this to be skipped, as it won't actually render as dark in Showkase.